packages/amplify-category-storage/resources/overrides-resource/S3/package.json

```diff
@@ -0,0 +1,17 @@
+{
+    "name": "overrides-dependencies",
+    "version": "1.0.0",
+    "description": "",
+    "scripts": {
+      "build": "tsc",
+      "watch": "tsc -w",
+      "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "dependencies": {
+      "@aws-amplify/cli-extensibility-helper": "1.0.1-ext19.0"
```

Comment:
This doesn't look right

Reply:
This is a sample skeleton sample package.json.
This can be changed to ^1.0.0 only once we do the first release for @aws-amplify/cli-extensibility-helper which will go out with our first release.